thespian/system/transport/TCPTransport.py

```diff
@@ -882,10 +881,8 @@ def _runWithExpiry(self, incomingHandler):
 
             wrecv = list(filter(None, wrecv))
             wsend = list(filter(None, wsend))
-            wrecv.extend(list(filter(None,
-                                     [I
-                                      for I in self._incomingSockets
-                                      if not self._incomingSockets[I].backoffPause()])))
+            wrecv.extend([ I for I in self._incomingSockets if I and
```

Comment:
Hm.. perhaps those assignments above could be changed into:
```
wrecv = [ h for h in wrecv if h ]
wsend = [ h for h in wsend if h ]
```
I had to go look up what `None` did when passed as the filter function; this would be clearer, you think?